File: benchmarks/file_io/walkdir.nim
import std/[times, os]
let home = getHomeDir()
let root = home / ".naab" / "language" / "src"
let start = cpuTime()
var count = 0
for path in walkDirRec(root):
  count += 1
echo int((cpuTime() - start) * 1_000_000)

File: benchmarks/file_io/walkdir.go
package main

import (
	"fmt"
	"os"
	"path/filepath"
	"time"
)

func main() {
	home, _ := os.UserHomeDir()
	root := filepath.Join(home, ".naab/language/src")
	start := time.Now()
	count := 0
	filepath.Walk(root, func(_ string, _ os.FileInfo, _ error) error {
		count++
		return nil
	})
	fmt.Println(time.Since(start).Microseconds())
}
